video_xblock/static/bower.json

```diff
@@ -7,7 +7,7 @@
     "videojs-wistia": "raccoongang/videojs-wistia#playback-toggle-fix",
     "videojs-transcript": "0.8.0",
     "videojs-resolution-switcher": "85f1e51c8babefca2d670dbd0b21a6307dbb4d21",
-    "videojs-offset": "raccoongang/videojs-offset",
+    "videojs-offset": "raccoongang/videojs-offset#master",
```

Comment:
We use master from our repository. If we don't mention branch, videojs-offset can be downloaded from bower cache (without our fixes)